### PR TITLE
Add Skeleton in the wild

### PR DESCRIPTION
**Pull request by canack**
Hi, I was open a PR https://github.com/charmbracelet/bubbletea/pull/1113
I moved my changes here. 🍀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [redis-viewer](https://github.com/SaltFishPr/redis-viewer) - A Redis database browser.
 - [redis_tui](https://github.com/mat2cc/redis_tui) - A Redis database browser.
 - [schemas](https://github.com/dhth/schemas) - Lets you inspect postgres schemas in the terminal.
+- [skeleton](https://github.com/termkit/skeleton): a multi-tab (TUI) library based on Bubble Tea framework.
```
Comment (Delta456) [inline README.md:101]:
```suggestion
- [skeleton](https://github.com/termkit/skeleton) - A TUI multi-tab library based on Bubble Tea framework.
```

Comment (Delta456) [reply README.md:101]: The CI also fails due to this reason `101:50  List item link and description must be separated with a dash`

Comment (canack) [reply README.md]: Thank you for review 🍀

**Review by Delta456 — APPROVED** [review]
Good work!

**Pull request merged**